src/test/java/no/digipost/signature/client/security/OrganizationNumberValidationTest.java

```diff
@@ -0,0 +1,31 @@
+package no.digipost.signature.client.security;
+
+import no.digipost.signature.client.TestCertificates;
+import org.junit.jupiter.api.Test;
+
+import java.security.cert.X509Certificate;
+
+import static no.digipost.signature.client.security.CertificateChainValidation.Result.TRUSTED;
+import static no.digipost.signature.client.security.CertificateChainValidation.Result.UNTRUSTED;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static uk.co.probablyfine.matchers.Java8Matchers.where;
+
+public class OrganizationNumberValidationTest {
+
+    @Test
+    void trusts_Posten_Norge_SEID1_enterprise_certificate() {
```

Comment:
Skulle vi hatt en tilsvarende test for SEID2?

Reply:
Jepp, la det til her 8a89ac4